src/FSharpVSPowerTools/UI/GeneralOptionsControl.cs

```diff
@@ -181,6 +187,13 @@ private void GeneralOptionsControl_Load(object sender, EventArgs e)
             QuickInfoPanelEnabled = _optionsPage.QuickInfoPanelEnabled;
             LinterEnabled = _optionsPage.LinterEnabled;
             OutliningEnabled = _optionsPage.OutliningEnabled;
+            PeekDefinitionEnabled = _optionsPage.PeekDefinitionEnabled;
+
+            if (!_optionsPage.PeekDefinitionAvailable)
+            {
+                chbPeekDefinition.Enabled = false;
+                chbPeekDefinition.Text = chbPeekDefinition.Text + " (VS2015+ only)";
```

Comment:
This will keep adding VS2015+ suffix to the label each time we open General options dialog.